GoogleForCreators/web-stories-wp
# Editor: Add link to revisions

## Context

In Gutenberg there's a link to the revisions comparison screen in the document tab It would be nice to do the same in the story editor as well, maybe in the PublishPanel component (packages/wp-story-editor/src/components/documentPane/publish/publish.js) below the poster image / publisher logo setting



## Summary

Adds revisions link / icon to publish panel

<img width="377" alt="Screen Shot 2022-09-08 at 6 34 32 AM" src="https://user-images.githubusercontent.com/62242/189101357-d707a577-dc9d-4c3f-9ac0-148952942663.png">






## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

0. Enable improved autosaves experiment
1. Create a new story
2. Publish story
3. There should now be a link to the revisions page in the Document tab
4. Verify the link works as expected
5. Make changes to story & save changes
6. Verify the revision count increases and the link points to the revisions page for the latest revision

Note: improvements to the revisions page itself are not part of this ticket.

## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #85
